views/event_details.hbs

```diff
@@ -0,0 +1,25 @@
+<div class="event-details">
+  <div class="eventDetailsHeader">
+    <div>
```

Comment:
I made div for to dives in the header , I need this for the header and the a href .
the result should be in div alone , and the header in another div . I think